README.md

```diff
@@ -235,6 +235,27 @@ checkpointing so the job restarts near where it was interrupted.
     --workload xpk-test-workload --cluster xpk-test
     ```
 
+    This will only delete `xpk-test-workload` workload in `xpk-test` cluster. Deletion will only begin if you type `Y` at the prompt.
+
+*   Workload Delete (delete all training jobs in the cluster):
+
+    ```shell
+    python3 xpk.py workload delete \
+    --cluster xpk-test
+    ```
+
+    This will delete all the workloads in `xpk-test` cluster. Deletion will only begin if you type `Y` at the prompt for every workload.
```

Comment:
nit: to adjust the readme that users only need to type yes/y once at the prompt.

Reply:
Done!